src/main/java/it/pagopa/interop/probing/telemetry/writer/dto/TelemetryDto.java

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.interop.probing.telemetry.writer.dto;
+
+import java.util.UUID;
+import it.pagopa.interop.probing.telemetry.writer.util.EserviceStatus;
+import lombok.Data;
+
+@Data
+public class TelemetryDto {
```

Comment:
Every class need an interface

Reply:
We  will open a technical debt task in order to align every project